src/widgets/line/lv_line.h

```diff
@@ -70,6 +82,35 @@ void lv_line_set_y_invert(lv_obj_t * obj, bool en);
  * Getter functions
  *====================*/
 
+/**
+ * Get the pointer to the array of points.
+ * @param obj           pointer to a line object
+ * @return              const pointer to the array of points
+ */
+const lv_point_precise_t * lv_line_get_points(lv_obj_t * obj);
+
+/**
+ * Get the number of points in the array of points.
+ * @param obj           pointer to a line object
+ * @return              number of points in array of points
+ */
+uint32_t lv_line_get_point_count(lv_obj_t * obj);
+
+/**
+ * Get the mutability of the stored point array pointer.
+ * @param obj           pointer to a line object
+ * @return              true: the point array pointer is mutable, false: constant
+ */
+bool lv_line_get_point_array_is_mutable(lv_obj_t * obj);
```

Comment:
What about this?
```suggestion
bool lv_line_is_point_array_mutable(lv_obj_t * obj);
```

Having both `get` and `is` a little bit stage to me. I vote for keeping `is` as there is no direct `set` counterpart.